slides/design-burst-week7.html

```diff
@@ -0,0 +1,185 @@
+      <div class="slides">
+        <section>
+          <img src="../assets/logos/logo-mobile.png" class="img-s" />
+          <h2>Design burst</h2>
+          <h3>Week 7: Error message design</h3>
+          <aside class="notes">
+            Sources: https://cxl.com/blog/error-messages/
+          </aside>
+        </section>
+        <section>
+          <section>
+            <h3><span class="bigunderline">1. Set clear expectations</span></h3>
+            <img src="https://i.imgur.com/50HVp96.png" />
+            <aside class="notes">
+              Source: https://material.io/components/text-fields/#anatomy
+            </aside>
+          </section>
+          <section>
+            <p>
+              Use helper text to convey additional guidance about form input
+              fields
+            </p>
+            <img src="https://i.imgur.com/Gsi1vNw.png" />
+          </section>
+          <section>
+            <p>
+              When text input isn’t accepted, an error message can display
+              instructions on how to fix it
+            </p>
+            <img src="https://i.imgur.com/axfy8KS.png" />
+            <p>
+              Use icons to flag issues for colorblind users
+            </p>
+          </section>
+          <section>
+            <p>
+              Give real time feedback to help users avoid making errors
+            </p>
+            <img src="https://i.imgur.com/bePjW17.png" />
+          </section>
+        </section>
+        <section>
+          <section>
+            <h3>
+              <span class="bigunderline">2. Craft helpful copy</span>
+            </h3>
+            <br />
+            <p>Describe what went wrong, why, and how to fix it</p>
+            <img src="https://i.imgur.com/AYEJJ1t.png" />
+            <aside class="notes">
+              Sources:
+              <ul>
+                <li>
+                  https://www.slideshare.net/hanochaloni/bad-error-messages
+                </li>
+                <li>
+                  https://medium.com/@toddohanian/10-usability-heuristics-for-user-interfaces-in-web-design-c179aa39b54e
+                </li>
+              </ul>
+              <p>
+                Amazon promo code entry input is an example of what not to do:
+                https://twitter.com/OptimiseOrDie/status/673957685742592000?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E673957685742592000&ref_url=https%3A%2F%2Fcxl.com%2Fblog%2Ferror-messages%2F
+              </p>
+            </aside>
+          </section>
+          <section>
+            <p>Don't blame the user</p>
+
+            <img src="https://i.imgur.com/H54fCv2.png" />
+            <aside class="notes">
+              Source:
+              https://uxmovement.com/forms/how-to-make-your-form-error-messages-more-reassuring/
+            </aside>
+          </section>
+          <section>
+            <p>Humour <i>can</i> help</p>
+            <img src="https://i.imgur.com/eGCUyVf.png" />
```

Comment:
I'm really not sure about humour in error messages. Users are often frustrated and annoyed trying to accomplish a task that's important to them, and errors are exactly the wrong time to inject humour/brand personality

https://twitter.com/cherrikissu/status/972524442600558594